public/components/bulkReindexController.react.js

```diff
@@ -37,7 +37,7 @@ export default class BulkReindexControllerComponent extends React.Component {
     }
 
     componentDidMount() {
-        let intervalPeriod = 1000;
+        let intervalPeriod = 5000;
```

Comment:
Why did you decide to extend this?

Reply:
To reduce the amount of calls to the DynamoDB table. It didn't seem necessary to be calling every second.

Reply:
Note for later: this is typically a property passed to the component and defined upstream.